httpbp/prometheus.go

```diff
@@ -0,0 +1,108 @@
+package httpbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/reddit/baseplate.go/prometheusbp"
+)
+
+const (
+	methodLabel           = "http_method"
+	successLabel          = "http_success"
+	codeLabel             = "http_response_code"
+	remoteServerSlugLabel = "http_slug"
```

Comment:
similarly we might want to remove the `remote-` prefix from the name here.

Reply:
I thought I caught them all. Those darm "remote" and "local" keep popping up. Let me remove this one too.